wallets/rn_cli_wallet_068_5/.env copy.example

```diff
@@ -0,0 +1,2 @@
+ENV_PROJECT_ID=XXXX
```

Comment:
Is this file still needed?

Reply:
yes this is for a separate repo / needed to be adjusted